src/sentry/integrations/tasks/sync_status_inbound.py

```diff
@@ -213,6 +225,21 @@ def sync_status_inbound(
         "integration_id": integration_id,
     }
     if action == ResolveSyncAction.RESOLVE:
+        # Check if the group was recently resolved and we should skip the request
+        # Avoid resolving the group in-app and then re-resolving via the integration webhook
+        # which would override the in-app resolution
+        recently_resolved_groups = []
+        for group in affected_groups:
+            if group.status == GroupStatus.RESOLVED and group_was_recently_resolved(group):
+                recently_resolved_groups.append(group)
+
+        affected_groups = [
+            group for group in affected_groups if group not in recently_resolved_groups
+        ]
```

Comment:
Super tiny nit, totally optional: this is a bit cleaner if you invert the check and append to a different list when a group has not been resolved recently. I'm not sure how many possible groups a single status inbound task can handle, but I imagine it won't make much of a practical difference.